ceph_deploy/mds.py

```diff
@@ -138,6 +138,11 @@ def mds_create(args):
                 distro.release,
                 distro.codename
             )
+            if distro.normalized_name == 'centos':
```

Comment:
This doesn't look right.  It should be ``normalized_name == 'redhat'``, but even that isn't enough.

It's not that we don't want to use MDS on RHEL, it's that we don't want to use MDS on RHEL + RHCS.  If you're using upstream packages, MDS will work just fine.